.prow/applications-catalog.yaml

```diff
@@ -0,0 +1,520 @@
+presubmits:
+  - name: pre-kubermatic-appcatalog-argocd
+    run_if_changed: "(pkg/ee/default-application-catalog/|.prow/)"
```

Comment:
Can we make this run only if the YAML file for the app in specific changes? If we update any app, it would trigger _all_ app tests, which seems overkill.